cypress/integration/26-register_registration/creator_navigates_to_description_tab/description_tab.js

```diff
@@ -119,3 +120,28 @@ When('they click the Remove Project icon', () => {
 Then('they see the Project is removed from the list of selected Projects', () => {
   cy.get('[data-testid^=project-chip]').should('not.exist');
 });
+
+// Scenario: Creator opens dropdown with Allowed Vocabularies
+And('their Institution has a Vocabulary set as "Allowed"', () => {});

```

Comment:
denne testen mangler egentlig det viktigste, nemlig å sette Vokabular til "Allowed". Hvis alle vokabular er satt til "Default" blir det en annen oppførsel igjen, så dette må sikkert mockes for testene. Kommer sikkert til å jobbe med frontenden for dette i neste uke.

Reply:
Ok